include/HubFramework/HUBViewModelBuilder.h

```diff
@@ -20,6 +20,7 @@
  */
 
 #import "HUBJSONCompatibleBuilder.h"
+#import <UIKit/UIKit.h>
```

Comment:
Same as for `HUBViewModel.h`, how about a forward class declaration of `UINavigationItem`?